Update nav steps to app storage. (#1843)
saucelabs/sauce-docs: docs/mobile-apps/automated-testing/appium/quickstart.md

## Step 3: Upload the Sample App

> 1. [Download the iOS Real Device Swag Labs App](https://github.com/saucelabs/my-demo-app-rn/releases/download/v1.3.0/iOS-Real-Device-MyRNDemoApp.1.3.0-162.ipa).

Reviewer: ```suggestion
1. [Download the iOS Real Device Swag Labs App](https://github.com/saucelabs/sample-app-mobile/releases/download/2.7.1/iOS.RealDevice.SauceLabs.Mobile.Sample.app.2.7.1.ipa).
```

2. Upload it to your Sauce Labs account, either through the Sauce Labs UI or by calling the API.

> - Sauce Labs UI: From [Sauce Labs Home > LIVE > Mobile App](https://app.saucelabs.com/live/app-testing), select **Upload App** and browse to find the `iOS-Real-Device-MyRNDemoApp.1.3.0-162.ipa` file you just downloaded.

Reviewer: Updates link to UI -- [App Management](https://docs.saucelabs.com/mobile-apps/live-testing/live-mobile-app-testing/#uploading-an-app) is now used to upload apps.

```suggestion
- Sauce Labs UI: From [Sauce Labs > App Management](https://app.saucelabs.com/app-management), select **Upload App** and browse to find the `iOS.RealDevice.SauceLabs.Mobile.Sample.app.2.7.1.ipa` file you just downloaded.
```

- API: Use the [Upload File to App Storage](/dev/api/storage/#upload-file-to-app-storage) API request to upload the sample app to your account.
  ```title="Example API Upload Request"
  curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
  --request POST 'https://api.us-west-1.saucelabs.com/v1/storage/upload' \

>   --form 'payload=@"/Users/lindsaywalker/Git/Example_Tests/iOS-Real-Device-MyRNDemoApp.1.3.0-162.ipa"' \
>   --form 'name="iOS-Real-Device-MyRNDemoApp.1.3.0-162.ipa"'

Reviewer: ```suggestion
  --form 'payload=@"/Users/lindsaywalker/Git/Example_Tests/iOS.RealDevice.SauceLabs.Mobile.Sample.app.2.7.1.ipa"' \
  --form 'name="iOS.RealDevice.SauceLabs.Mobile.Sample.app.2.7.1.ipa"'
```
